conductor_api/src/conductor/admin.rs

```diff
@@ -381,22 +388,73 @@ impl ConductorAdmin for Conductor {
         Ok(())
     }
 
-    fn add_agent(&mut self, new_agent: AgentConfiguration) -> Result<(), HolochainError> {
+    fn add_agent(
+        &mut self,
+        id: String,
+        name: String,
+        holo_remote_key: Option<&str>,
```

Comment:
```suggestion
        holo_remote_key: Option<String>,
```

It's getting converted to a string later anyway